include/zephyr/bluetooth/conn.h

```diff
@@ -454,6 +529,66 @@ int bt_conn_get_remote_info(struct bt_conn *conn,
 int bt_conn_le_get_tx_power_level(struct bt_conn *conn,
 				  struct bt_conn_le_tx_power *tx_power_level);
 
+#if defined(CONFIG_BT_TRANSMIT_POWER_CONTROL)
+/** @brief Get enhanced connection transmit power level.
+ *
+ *  @param conn           Connection object.
+ *  @param tx_power_level Transmit power level descriptor.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @return -ENOBUFS HCI command buffer is not available.
+ */
+int bt_conn_le_enhanced_get_tx_power_level(struct bt_conn *conn,
+				struct bt_conn_le_tx_power *tx_power);
```

Comment:
I'll make this comment just here, but it applies to all places where you have split lines: the style for split line alignment in the Zephyr Bluetooth code is N tabs + 0-7 spaces, so that the line starts right after the opening parenthesis of the first line. Most editors can be configured to do this automatically, so you shouldn't have to think about it.